sqladmin/models.py

```diff
@@ -727,6 +725,9 @@ def _build_url_for(self, name: str, request: Request, obj: Any) -> URL:
             pk=get_object_identifier(obj),
         )
 
+    def _get_prop_name(self, prop: MODEL_ATTR) -> str:
```

Comment:
You're using that code in many other places, consider replacing them as well?

e.g. 
<img width="310" alt="Screenshot 2024-02-17 at 18 30 29" src="https://github.com/aminalaee/sqladmin/assets/144918/23441024-db3e-46b5-b976-436d484d4809">

Reply:
Thanks for pointing this out, as far as I checked none of them were dealing with this case and they were checking other conditions.